k9mail/src/main/java/com/fsck/k9/activity/MessageList.java

```diff
@@ -1435,6 +1435,8 @@ public void enableActionBarProgress(boolean enable) {
     public void displayMessageSubject(String subject) {
         if (mDisplayMode == DisplayMode.MESSAGE_VIEW) {
             mActionBarSubject.setText(subject);
```

Comment:
I'm just curious why we are ignoring the passed in `subject` in the else block. It feels wrong.